chain/client/src/stateless_validation/partial_witness/partial_witness_actor.rs

```diff
@@ -343,7 +347,7 @@ impl PartialWitnessActor {
         let encode_timer = metrics::PARTIAL_WITNESS_ENCODE_TIME
```

Comment:
`PARTIAL_WITNESS_ENCODE_TIME` metric no longer represents time to just encode. We can perhaps rename to `PARTIAL_WITNESS_ENCODE_AND_SEND_TIME`. Please quickly ping @pugachAG and let him know about this change in behavior.

Reply:
Hmm, is it worth specifying it? I guess it just adds a message to the channel on every iteration which should not be expensive

Reply:
Actually it can block if the channel is bounded and full but I am struggling to understand the macro to be sure